packages/shared-ui/src/components/Layout.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react'
+
+interface LayoutProps {
+  children: ReactNode
+  header?: ReactNode
+  footer?: ReactNode
+  backgroundImageSrc?: string
+}
+
+export function Layout(props: LayoutProps) {
+  return (
+    <div
+      style={{ backgroundImage: props.backgroundImageSrc }}
```

Comment:
I think we can include the `backgroundImageSrc` inside the shared ui lib, so it would be something like:

```
import SpaceImage from '../whatever/space.jpeg'

...

<div style={{ backgroundImage: `url(${SpaceImage})` }} />
```

Reply:
@spsjvc microbundle doesn't support exported images, we could include `space.jpeg` in shared-ui dist folder and pass it onto the `Layout` component. Thoughts?